cmd/root.go

```diff
@@ -61,12 +62,12 @@ func Execute() {
 	// the distribution parameters, which we accept as CLI flags as well
 	cmd.Flags().StringVar(&cfg.Distribution.ExeName, "name", "otelcol-custom", "The executable name for the OpenTelemetry Collector distribution")
 	cmd.Flags().StringVar(&cfg.Distribution.LongName, "description", "Custom OpenTelemetry Collector distribution", "A descriptive name for the OpenTelemetry Collector distribution")
-	cmd.Flags().StringVar(&cfg.Distribution.Version, "version", "1.0.0", "The version for the OpenTelemetry Collector distribution")
```

Comment:
This field is about the version to use for the underlying build. Instead of removing, this needs to have a new name, which will cause a breaking change.

Reply:
It is confusing and I somehow thought `otelcol-version` is the version of the underlying collector binary. I am still confused what is the `version` and `otelcol-version` being used for ?

Reply:
This repository builds custom distributions of the OpenTelemetry Collector, and the distribution itself might have its own version. Imagining that Jaeger would be such a distribution, `version` would be Jaeger's version (like 2.0.0-alpha1), and `otelcol-version` is which version of OpenTelemetry Collector version to base on (0.22.0, for instance)

Reply:
Thank you Juraci for the explanation. I think I get it now. This means we have to deal with 3 versions variables in total. In order to prevent making breaking change, I opted to go ahead with a subcommand: 
```
$ ./opentelemetry-collector-builder version
dev
```
Let me know in case you think we should do it in some other way.